src/utils/validators.js

```diff
@@ -27,7 +27,7 @@ export const TIKTOK_REGEX = ".com/@)([a-zA-Z0-9_-]+([/.])?)+$"
 // 3. The last section after "+" is to match the TLD segment of the domain
 // 4. The "^" and "$" ensures that we are matching the whole string (to prevent the user from adding "https://")
 export const DOMAIN_NAME_REGEX =
-  "^([a-z0-9]([a-z0-9-]{0,61}[a-z0-9])?\\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9]$"
+  "^([a-z0-9]([a-z0-9-]{0,61}[a-z0-9])?\\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9]/?$"
```

Comment:
regex101 seems to complain about the `/?`, saying that 
> / An unescaped delimiter must be escaped; in most languages with a backslash (\)
> ? The preceding token is not quantifiable

i think you might be missing a `\` as follows: `\/?`

Reply:
I realised it was being automatically removed by prettier, so I had to add a double escape character. Thanks for the catch!